source/release-notes-ps-v8.0.28.upd.rst

```diff
@@ -0,0 +1,32 @@
+.. _pdps.8.0.28.upd:
+
+********************************************************************************
+|pdmysql| 8.0.28 using Percona Server for MySQL Update (|date|)
+********************************************************************************
+
+:Date: |date|
+:Installation: :ref:`installation`
+
+|pdmysql| is a single solution with the best and most critical enterprise components from the |mysql| open-source community, designed and tested to work together. 
+
+This update of |pdmysql| using |PS| is based on `Percona Server for MySQL 8.0.28-20 <https://www.percona.com/doc/percona-server/8.0/release-notes/Percona-Server-8.0.28-20.html>`_. It also includes the updated version of `Percona XtranBackup 8.0.28-21 <https://docs.percona.com/percona-xtrabackup/latest/release-notes/8.0/8.0.28-21.0.html>`_.
+
+Release Highlights
+=================================================
+
+* Encryption functions and variables to manage the encryption range are added. The functions may take an algorithm argument. Encryption converts plaintext into ciphertext using a key and an encryption algorithm.
```

Comment:
Would it be possible to link to the documentation of this feature here?

Reply:
Added